### PR TITLE
Added feature to claim issues from Slack

### LINKED ISSUE #41: Claiming issues from Slack
## Description
As a user
I need to claim issues from slack
so that I can automate issue claiming.

### Update [Required]
- [ ] Add functionality to claim an issue from Slack.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Added feature to claim issues from Slack via slash command /sysbot_claim

Fixes #41 

# Type of Change:

- Code
- New feature (a non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?

before_command
after_command

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

File: main_server.py [hunk 1]
```diff
@@ -97,6 +97,14 @@ def slack_assign_receiver():
         return Response(status=200)
 
 
+@app.route('/claim', methods=['POST'])
+def slack_claim_receiver():
+    if request.headers['Content-Type'] == 'application/x-www-form-urlencoded':
+        data = request.form
```
Comment (ramitsawhney27) [inline main_server.py]: Add a more suitable name for data, that can explain its meaning to anyone who sees the code. Maybe we can start commenting in depth to explain each line to focus on more readability.

File: slack_functions.py [hunk 2]
```diff
@@ -72,11 +72,6 @@ def approve_issue_label_slack(data):
...
 
 
-def send_message_to_channels(channel_id, message):
-    body = {'username': 'Sysbot', 'as_user': True, 'text': message, 'channel': channel_id}
-    requests.post(dm_chat_post_message_url, data=json.dumps(body), headers=headers)
-
-
 def check_newcomer_requirements(uid):
     body = {'user': uid, 'include_labels': True}
     r = requests.post(get_user_profile_info_url, data=body, headers=headers_legacy_urlencoded)
```
Comment (ramitsawhney27) [inline slack_functions.py]: What is r? Why is it being assigned a value if it’s not going to be used in the future?

File: slack_functions.py [hunk 3]
```diff
@@ -127,6 +122,35 @@ def assign_issue_slack(data):
...
 
 
+def claim_issue_slack(data):
+    params = data.get('text','')
+    tokens = params.split(' ')
+    channel_id = data.get('channel_id','')
+    if params != '' and len(tokens) == 3:
+        #The tokens are issue number, repo name, and claimant's username
+        status = issue_assign(tokens[1], tokens[0], tokens[2], 'systers')
+        print status
```
Comment (ramitsawhney27) [inline slack_functions.py]: Possibly comment the status prining out now?

File: slack_functions.py (hunk 2 above)
Comment (sammy1997) [reply slack_functions.py]: I am using r as response. I have extracted r.json() in the next line.

**Pull request merged**